datetimeformat.html

```diff
@@ -91,11 +93,13 @@ <h1>BasicFormatMatcher ( _options_, _formats_ )</h1>
             1. If _format_ has a field [[&lt;_property_&gt;]], let _formatProp_ be _format_.[[&lt;_property_&gt;]]; else let _formatProp_ be *undefined*.
             1. If _optionsProp_ is *undefined* and _formatProp_ is not *undefined*, decrease _score_ by _additionPenalty_.
             1. Else if _optionsProp_ is not *undefined* and _formatProp_ is *undefined*, decrease _score_ by _removalPenalty_.
+            1. <ins>Else if _property_ is *"timeZoneName"*, then</ins>
+              1. <ins>If _optionsProp_ is *"short"* or *"shortGeneric"* and _formatProp_ is *"shortOffset"*, decrease _score_ by _offsetPenalty_.</ins>
```

Comment:
I think you need to handle the case when _optionsProp_ and _formatProp_ are equal, too.

Reply:
when optionsProp and formatProp are equal, why should we change the score?